kpi/tests/api/v2/test_api_service_usage.py

```diff
@@ -227,13 +181,12 @@ def __update_xform_counters(self, asset: Asset, submissions: int = 0):
                     date=today.date(),
                     counter=submissions,
                     xform=self.xform,
-                    user_id=asset.owner_id,
+                    user_id=asset.owner.id,
                 )
             )
-            print(vars(self.counter))
             self.counter.save()
 
-    def __expected_file_size(self):
+    def expected_file_size(self):
```

Comment:
:+1: yeah personally I would limit _ prefix usage unless there's a good reason to have these pretend private functions. It's not like this class is getting used heavily and extended for reuse.